vendor/brave-ios/BUILD.gn

```diff
@@ -65,6 +65,7 @@ ios_framework_bundle("brave_rewards_ios_framework") {
     "//sql",
     "//url",
     "//components/os_crypt",
+    "//brave/components/ipfs",
```

Comment:
Is this supposed to be added for this PR?

Reply:
this commit just to run it on CI, it will be removed before merge